pkg/registry/storage/rest/storage_storage.go

```diff
@@ -79,7 +79,7 @@ func (p RESTStorageProvider) v1beta1Storage(apiResourceConfigSource serverstorag
 	storage := map[string]rest.Storage{}
 
 	// register csistoragecapacities
-	if resource := "csistoragecapacities"; apiResourceConfigSource.ResourceEnabled(storageapiv1beta1.SchemeGroupVersion.WithResource(resource)) {
+	if resource := "csistoragecapacities"; apiResourceConfigSource.ResourceEnabled(storageapiv1.SchemeGroupVersion.WithResource(resource)) {
```

Comment:
this isn't correct... v1 storage is already configured on line 135 below:

```
	if resource := "csistoragecapacities"; apiResourceConfigSource.ResourceEnabled(storageapiv1.SchemeGroupVersion.WithResource(resource)) {
```

after 1.28 ships, the csistoragecapacities lines in v1beta1Storage can be dropped entirely